forward_pass_mocking.ipynb

```diff
@@ -0,0 +1,1446 @@
+{
```

Comment:
We don't add notebooks especially not here! Is there a particular reason for this one?